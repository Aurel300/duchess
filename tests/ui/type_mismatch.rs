duchess::java_package! { //~ ERROR: proc macro panicked
    //~^ HELP: generic type parameter `Id { data: "bool" }` not among in-scope parameters: []
    package java.lang;

    public class java.lang.Object {
        public java.lang.Object();
        public native bool hashCode();
    }
}
