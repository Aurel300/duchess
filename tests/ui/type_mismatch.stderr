error: proc macro panicked
 --> $DIR/type_mismatch.rs:1:1
  |
1 | / duchess::java_package! {
2 | |
3 | |     package java.lang;
4 | |
... |
8 | |     }
9 | | }
  | |_^
  |
  = help: message: generic type parameter `Id { data: "bool" }` not among in-scope parameters: []

error: aborting due to previous error

